packages/angular/src/generators/cypress-component-configuration/cypress-component-configuration.spec.ts

```diff
@@ -746,6 +746,7 @@ function getCmpsFromTree(
   tree: Tree,
   options: { basePath: string; name: string }
 ) {
+  console.log(tree.listChanges().map((c) => c.path));
```

Comment:
I assume this was for debug purposes? Should it be removed?

Reply:
🤦 yes lol. removing before merging